docs/features/explorer.md

```diff
@@ -139,6 +139,67 @@ Component.Explorer({
 })
 ```
 
+### Use `sort` with pre-defined sort order
+
+Here's another example where a map containing file/folder names is used to define the sort order of the explorer in quartz. All files/folders that aren't listed inside of `nameOrderMap` will appear at the top of that folders hierarchy level. It's also worth mentioning, that the smaller the number set in `nameOrderMap`, the higher up the entry will be in the explorer. Incrementing every folder/file by 100, makes ordering files in their folders a lot easier. Lastly, having the `mapFn` function before or after `sortFn` doesn't matter.
+
+```ts title="quartz.layout.ts"
+Component.Explorer({
+  sortFn: (a, b) => {
+    const nameOrderMap: Record<string, number> = {
+      "poetry-folder": 100,
+      "essay-folder": 200,
+      "research-paper-file": 201,
+      "dinosaur-fossils-file": 300,
+      "other-folder": 400,
+    }
+
+    let orderA = 0
+    let orderB = 0
+
+    if (a.file && a.file.slug) {
+      orderA = nameOrderMap[a.file.slug] || 0
+    } else if (a.name) {
+      orderA = nameOrderMap[a.name] || 0
+    }
+
+    if (b.file && b.file.slug) {
+      orderB = nameOrderMap[b.file.slug] || 0
+    } else if (b.name) {
+      orderB = nameOrderMap[b.name] || 0
+    }
+
+    return orderA - orderB
+  },
+})
+```
+
+For reference, this is how the quartz explorer window would look like with that example.
+
+```
+📖 Poetry Folder
+📑 Essay Folder
+    ⚗️ Research Paper File
+🦴 Dinosaur Fossils File
+🔮 Other Folder
+```
+
+And this is how the file structure would look like.
+
+```
+index.md
+poetry-folder
+    index.md
+essay-folder
+    index.md
+    research-paper-file.md
+dinosaur-fossils-file.md
+other-folder
+    index.md
+```
+
+As you can see `nameOrderMap` is using the raw file/folder name (without file extensions). If you would rather use the frontmatter name of the file/folder, then ... **(ToDo: how does one modify the code so the `nameOrderMap` can be filled with the frontmatter name?)**
```

Comment:
There's still a remaining TODO here, is this sentence intentional? (should probably resolve TODO or remove sentence if its just a leftover)

Reply:
The "..." with the ToDo description was what I was talking about yesterday.

"would you be so kind and add that, or alternatively we could also leave away that last part entirely, as it's not good practice to reference frontmatter names in code anyway."